app/assets/stylesheets/home.scss

```diff
@@ -0,0 +1,49 @@
+@import "foundation_and_overrides";
+
+body {
+  background: url("Home_photo.jpg") no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
+
+.footer {
+  background: transparent;
+  text-align: center;
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  height: 4rem;
+  padding: 0.5rem;
+}
+  .footer .links a {
+    border: 1px dotted white;
+    color: white;
+    font-weight: bold;
+    margin: 5rem;
+    padding: 1rem;
+    text-align: center;
+  }
+
+.top-bar ul li {
+  font-weight: bold;
+}
+
+.top-bar-section ul li {
+    background: transparent;
+    margin: 0.5rem 2rem;
+    font-weight: bold;
+    border: 1px dotted white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces